syntax = "proto3";
//
service Greeter {
  // Get the binary of the file requested if it exists else return an error
  rpc getFile (fileRequest) returns (fileBytes) {}
}

// Mesage with the file path to request
message fileRequest {
  string file_path = 1;
}

//Message with the byte array of the file requested
message fileBytes {
  bytes file_bytes = 1;
}


//compilation: python -m grpc_tools.protoc -I./gRPC --python_out=./Common --pyi_out=./Common --grpc_python_out=. ./gRPC/p2p.proto